develop/parachains/intro-polkadot-sdk.md

```diff
@@ -54,6 +54,32 @@ Every blockchain platform relies on a decentralized network of computers—calle
 
 FRAME provides the core modular and extensible components that make the Substrate SDK flexible and adaptable to different use cases. FRAME includes Rust-based libraries that simplify the development of application-specific logic. Most of the functionality that FRAME provides takes the form of plug-in modules called [pallets](/polkadot-protocol/glossary#pallet){target=\_blank} that you can add and configure to suit your requirements.
 
+``` mermaid
+graph TB
+    subgraph sg1[Client/Host]
+        direction LR
+        A[JSON-RPC]
+        B[libp2p]
+        C[Transaction Pool]
+        D[Block Builder]
+        E[Database]
+        F[Host Functions]
+    end
+    subgraph sg2[Runtime]
+        direction LR
+        G[Runtime API]
+        H[Executive]
```

Comment:
have you explained "executive" anywhere? I worry this might be some low level of detail that the old docs had and we are mistakenly copying it. 

For any new keyword added, ask yourself, does the reader already know what this is? if yes, good. If no, either remove, or add a link to where it is explained. 

Executive is a fairly low level detail of FRAME. 

I have some better resources for you to create a more informed version of this diagram: 

- https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#summary
- https://polkadot-blockchain-academy.github.io/pba-content/singapore-2024/syllabus/6-Polkadot-SDK/FRAME/1-Intro_to_FRAME/Intro_to_FRAME-slides.html#/5